scripts/ai-help-macros.ts

```diff
@@ -72,14 +80,11 @@ export async function updateEmbeddings(
       });
     } catch (e: any) {
       const {
-        data: {
-          error: { message, type },
-        },
+        error: { message, type },
         status,
-        statusText,
       } = e.response;
```

Comment:
I had to change `e.response` to `e` here, otherwise the error could not be destructured properly.